src/components/Switch.jsx

```diff
@@ -46,13 +48,27 @@ module.exports = React.createClass({
                        onExit={ this.close } />)
   },
 
+  hasMaxChildren() {
+    let { app, parent } = this.props
+
+    if (!parent) return false
```

Comment:
OK. I'll need to run a couple things we're doing in NTHP by you to see if the best approach is to further update CK or just change my block definitions.